CedarJavaFFI/src/objects.rs

```diff
@@ -235,7 +235,7 @@ impl<'a> JEntityId<'a> {
 
     /// Decode the object into its string representation
     pub fn get_string_repr(&self) -> String {
-        self.id.to_string()
+        self.id.as_ref().to_string()
```

Comment:
Why not `.escaped()`? That would have been the original behavior, right?

Reply:
True, and I can change it.  I thought that perhaps folks manipulating the string in Java might expect to manipulate the unescaped string.  But I suppose no one has complained about the current behavior

Reply:
No strong preference, I was just wondering if the behavior change was intentional. I think both options are reasonable, but might be better stick with the status quo 🤷‍♀️

Reply:
Changed to `.escaped()`